src/app/components/form/form.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+
+interface Mentor {
+  value: string;
```

Comment:
What's the difference between value and viewValue? Perhaps fields could be renamed to reflect it better?

Reply:
I don't know, I just started to learn Angular Material and used example 
![image](https://user-images.githubusercontent.com/78438345/110966132-a3e7b280-835d-11eb-95c9-64b605fa26f8.png)

Reply:
but this is just for demo version now